runtime/compiler/env/j9methodServer.hpp

```diff
@@ -266,15 +265,9 @@ class TR_ResolvedRelocatableJ9JITaaSServerMethod : public TR_ResolvedJ9JITaaSSer
    virtual void *                methodHandleConstant(int32_t cpIndex) override;
    virtual bool                  isUnresolvedMethodHandle(int32_t cpIndex) override;
    /* No need to override the stringConstant, fieldAttributes and staticAttributes method as the parent method will be sufficient */
-   virtual int32_t               virtualCallSelector(uint32_t cpIndex) override { TR_ASSERT(0, "called");  return 0; }
-   virtual char *                fieldSignatureChars(int32_t cpIndex, int32_t & len) override { TR_ASSERT(0, "called");  return NULL; }
-   virtual char *                staticSignatureChars(int32_t cpIndex, int32_t & len) override { TR_ASSERT(0, "called");  return NULL; }
-   virtual TR_OpaqueClassBlock * classOfStatic(int32_t cpIndex, bool returnClassForAOT = false) override { TR_ASSERT(0, "called");  return NULL; }
-   virtual TR_ResolvedMethod *   getResolvedPossiblyPrivateVirtualMethod( TR::Compilation *, int32_t cpIndex, bool ignoreRtResolve, bool * unresolvedInCP) override { TR_ASSERT(0, "called");  return NULL; }
-   virtual bool                  getUnresolvedFieldInCP(int32_t cpIndex) override { TR_ASSERT(0, "called");  return false; }
```

Comment:
I see that getUnresolvedFieldInCP does not have a definition for a J9ResolvedMethod, but we have provided one for ResolvedJ9JITaaSServerMethod. We should probably move that definition to the Relocatable JITaaS version.

Reply:
I agree, moved `getUnresolvedFieldInCP` to the Relocatable JITaaS version.

Reply:
will do the same with `getUnresolvedStaticMethodInCP` and `getUnresolvedSpecialMethodInCP`

Reply:
However, is there any possibility that these 3 functions are actually used by `ResolvedJ9JITaaSServerMethod`?

Reply:
```
TR_ASSERT(c->compileRelocatableCode(),"isUnresolvedFieldInCP only callable in AOT compiles");
```
I guess these asserts mean that they should only be called in AOT compiles.

Reply:
From what I have seen these methods are only called from AOT related code